### ISSUE TITLE
Make templating dependency weak?

### ISSUE THREAD

**Issue by SachaG** [meteor-utilities/avatar#62]
I feel like we should revisit the React compatibility issues. Maybe a good start would be making the Blaze dependency weak?

**Comment by timothyarmes** [comment]
Yes, I've been thinking about it this week too. I don't know if it's best to have a separate project or to try to have one project do everything. 

It would certainly be a much cleaner implementation if it were React only, but that severely affects maintenance.
